FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DHCP.c

```diff
@@ -627,7 +627,7 @@ static void prvInitialiseDHCP( void )
 	}
 	else
 	{
-		/* There was a problem with the randomiser. */
+		/* There was a problem with the randomizer. */
```

Comment:
This is changing from English to American spelling – our dictionary is probably using American spelling.